packages/raja/package.py

```diff
@@ -191,6 +194,8 @@ def initconfig_package_entries(self):
 
         entries.append(cmake_cache_option("RAJA_ENABLE_VECTORIZATION", "+vectorization" in spec))
 
+        entries.append(cmake_cache_option("RAJA_ENABLE_OPENMP_TASK", "+omptask" in spec))
```

Comment:
Does `omptask` need to be added as a variant?

Reply:
D'oh! Missed that. Thank you.